content/components/information/contributing.md

```diff
@@ -6,7 +6,7 @@ excludeFromStatus: true
 ## Github issues
 We’ll be using Github issues to keep track of new component submissions, bugs, design feedback, and any other suggestions related to the design system. To help us understand the kind of contribution you want to make, we ask that you first [submit an issue on GitHub](https://github.com/indiana-university/rivet-source/issues/new/choose).
 
-{{< button url="https://github.com/indiana-university/rivet-source/issues/new/choose" >}}Create an issue{{< /button >}}
+{{<button  type="button" url="https://github.com/indiana-university/rivet-source/issues/new/choose" >}}Create an issue{{< /button >}}
```

Comment:
You don't need the type="button" attribute here. This is actually a Hugo shortcode called `{{< button >}}` that outputs a link style with the `.rvt-button` class.

Sorry, that's a bit confusing.

Reply:
Ahhh, that makes sense. Thanks for the heads up. 🙂